rfc11720

```diff
@@ -104,6 +153,40 @@ The objectives of the myTeams project RFC is to allow communication between myTe
     - Invalid format
 - 24
     - Resource already exists
+- 25
+    - Client does not log/exist in database
+
+
+- 30
+    - Unknown team
```

Comment:
14: Not found ?

Reply:
Nop i wan't to know what is not found :D